src/adapters/database/DbInterface.h

```diff
@@ -0,0 +1,27 @@
+#ifndef SRC_DBINTERFACE_H
+#define SRC_DBINTERFACE_H
+
+#include <string>
+#include <set>
+#include "../../application/AbstractDatabaseInterface.h"
+#include "AbstractSqlDatabase.h"
+#include "../../application/DatabaseColumn.h"
+
+namespace Adapters::Database
+{
+    class DbInterface : public AbstractDatabaseInterface
```

Comment:
Possibly rename to `DatabaseInterface` so it's in line with `AbstractDatabaseInterface`